src/components/box/box.component.tsx

```diff
@@ -52,87 +59,70 @@ export interface BoxProps
   rowGap?: Gap;
   /** Design Token for Box Shadow. Note: please check that the box shadow design token you are using is compatible with the Box component. */
   boxShadow?: BoxShadowsType;
+  /** @private @ignore */
+  className?: string;
+  /** Set the color attribute of the Box component */
+  color?: string;
```

Comment:
It think that after adding `color`, `bg`, `backgroundColor` and `opacity` props explicitly you can remove the `interface BoxProps extends ColorProps`

Reply:
Fixed